contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+
+### Free
+
+One of the benefits (and reasons) for SQL's popularity is that there is free software available (%like MySQL, PostgreSQL and SQLite%) as well as paid ones (%%like Microsoft SQL Server and Oracle%%) that come with more functionality.
+
+One such example is MySQL. It is free of cost and comes with the official [MySQL Workbench](https://dev.mysql.com/doc/workbench/en/) as well as other third party easy-to-use <tooltip content="Graphical User Interface">GUIs</tooltip>, which are less daunting to new users as compared to a <tooltip content="Command Line Interface">CLI</tooltip>.
+
+### Highly Compatible
+
+SQL software works on many operating systems (%%like Windows, Mac OS, Ubuntu, Red Hat among others%%) and more importantly, can be integrated with many languages. This includes programming languages like C++, Ruby, Java, Python, JavaScript (Node.js), and Go.
+
+In addition, online playgrounds like [DB Fiddle](https://www.db-fiddle.com/) and [JDoodle](https://www.jdoodle.com/execute-sql-online) make it easy to use and learn the SQL language quickly without the hassle of setting up an environment or application.
+
+## Disadvantages
+
+SQL is not without its problems. In general, the biggest problem is with regards to the features of SQL.
+
+Although SQL databases use established <tooltip content="American National Standard Institutes">ANSI</tooltip> & <tooltip content="International Organization for Standardization">ISO</tooltip> standards, some SQL software systems (%%PostgreSQL, for example%%) add proprietary extensions to standard SQL.
+Due to this, the available feature set can vary according to what you're using. This can make SQL confusing and frustrating to use when switching SQL software.
+
+Apart from that, most **SQL** problems are not uniformly applicable across all SQL software. For example, MySQL suffers from concurrency issues. Though it performs well with read operations, it *can* be problematic when there are many concurrent read-write operations. A symptom of this issue would be a sudden slowdown of a query. However, PostgreSQL deals with concurrency well by implementing <tooltip content="each query transaction is isolated from other transactions running simultaneously in the system">transactions isolation</tooltip>.
+
+---
+
+## How to get started with SQL?
+
+The first thing to do when getting started with SQL is to choose a SQL software. You can either just use an online SQL playground or install a dedicated application. Our suggestion is to use either MySQL or SQLite online.
+
+SQL has been around for a long time and hence, you can find really good books and resources to learn it well. However, these resources can be overwhelming. So, you can decide which parts you want to learn, as you will likely not need to know everything to get started with using a SQL database.
```

Comment:
The first sentence is repeated earlier in the article. Try to avoid repetition given that this is a short article.